src/lib.rs

```diff
@@ -14,26 +14,32 @@ use tree_sitter::Query;
 
 mod args;
 mod language;
+mod line_buffer;
+mod lines;
 mod macros;
 mod matcher;
 mod plugin;
 mod printer;
 mod project_file_walker;
+mod query_context;
 mod searcher;
+mod sink;
 mod treesitter;
+mod use_printer;
```

Comment:
...and so then to avoid a namespace conflict used the (maybe questionable) naming convention of `use_*` for "our configuration/usage of ___"